### PR TITLE
Update outdated identifier %i with %d

### PR DESCRIPTION
**Pull request by nmklong**
The identifier `%i` is outdated and needs to be updated with `%d`

### CODE REVIEW

**Comment by derekrobbins** [comment]
it looks like this has been addressed in the source repo. Can we pull down the changes from the source? 

https://github.com/bobthecow/mustache.php/blob/main/src/Mustache/Engine.php

**Comment by nmklong** [comment]
> it looks like this has been addressed in the source repo. Can we pull down the changes from the source?
> 
> https://github.com/bobthecow/mustache.php/blob/main/src/Mustache/Engine.php

@derekrobbins 
It was addressed in `v2.12.0` (we forked from `v2.6.0`, with a small adjustment, see https://github.com/Mindtouch/mustache.php/commit/9b70a7a836ac509b6c81568aafc802c41d248165). 

However together with that fix in `v2.12.0`, they also introduced a lot of other changes as well. For example if I copy the content, there will be errors like these:

Screenshot_20230123_043528

Also the problem in https://github.com/MindTouch/Deki/issues/22583 arose from the new features introduced in version [v2.7.0](https://github.com/bobthecow/mustache.php/releases/tag/v2.7.0), so we may just want to stick with version `v2.6.0` until we determine what those tests are protecting as mustache seems to be just doing what it's supposed to do. At first I thought it was for the purpose of having `{{random_variable_name}}` in the content but looks like even with our current version that is not possible to do (it will throw an error when displaying the content), also I don't think if we have any customer using mustache's syntax in the content

So in short, just replacing the outdated identifier with the correct one seems to be the safest way.

**Review by derekrobbins — APPROVED** [review]
